java/compaction/compaction-core/src/test/java/sleeper/compaction/testutils/InMemoryCompactionJobStatusStoreTest.java

```diff
@@ -484,7 +511,7 @@ private CompactionJob addFinishedJob(Instant createdTime, RecordsProcessedSummar
     private CompactionJob addFinishedJobUncommitted(Instant createdTime, RecordsProcessedSummary summary, String taskId) {
         CompactionJob job = addStartedJob(createdTime, summary.getStartTime(), taskId);
         store.fixUpdateTime(defaultUpdateTime(summary.getFinishTime()));
-        store.jobFinished(compactionJobFinished(job, summary).taskId(taskId).committedBySeparateUpdate(true).build());
+        store.jobFinished(compactionJobFinished(job, summary).taskId(taskId).build());
```

Comment:
This method is now identical to the version above. Please check the tests that use them and just keep one or the other.